src/Core/Components/DataGrid/Columns/ColumnBase.razor.cs

```diff
@@ -36,10 +37,14 @@ public abstract partial class ColumnBase<TGridItem>
     [Parameter] public Align Align { get; set; }
 
     /// <summary>
-    /// If true, generate a title attribute for the cell contents
+    /// If true, generate a title and aria-label attribute for the cell contents
     /// </summary>
     [Parameter] public bool Tooltip { get; set; } = false;
 
+    /// <summary>
+    /// Defines the value to be used as the tooltip and aria-label in this column's cells
+    /// </summary>
+    [Parameter] public Expression<Func<TGridItem, string?>>? TooltipText { get; set; }
```

Comment:
Why did you use an `Expression` and not `Func<TGridItem, string?>`?

In my opinion, most of the time you'll need Func or Action if you're simply executing code. You need Expression when the code needs to be analyzed, serialized or optimized before being executed. Expression is used to think the code, Func/Action to execute it.

Reply:
I'm using the same construct that was already available in the `PropertyColumn` for the `Property` parameter. Short answer: it is faster. See also https://stackoverflow.com/questions/793571/why-would-you-use-expressionfunct-rather-than-funct?rq=2

Reply:
I'm not sure that `Expression` is faster than `Func`. Probably for parsing the tree when you need to generate an SQL query, but not for executing a simple method like `c => c.Name`. We don't need to change this now, but I found the compiled code more complex